understanding/20/error-identification.html

```diff
@@ -99,7 +97,7 @@ <h2>Intent of Error Identification</h2>
             In other cases, it may be more appropriate to show errors inline, with error messages next the specific
             fields that are in error. Errors could also be listed in an alert, or dialog. This criterion does not
             cover which of these methods should be used - the only requirement is for errors (which are not already
-            clear from the specific nature/context of the input fields) to be presented to users in text.
+            clear from the specific nature/context of the input fields) to be presented to users in text or alt text.
```

Comment:
replace "alt text" with "a text alternative for non-text content"

Reply:
done (with a bit more wordsmithing to make it flow better)